pkg/serverless/trace/trace.go

```diff
@@ -37,6 +38,11 @@ func (l *LoadConfig) Load() (*config.AgentConfig, error) {
 // Start starts the agent
 func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load) {
 	if enabled {
+		// during hostname resolution the first step is to make a GRPC call which is timeboxed to a 2 seconds deadline
+		// in the serverless mode, we don't start the GRPC server so this call will fail and cause a 2 seconds delay
+		// by setting cmd_port to 0, this will cause the GRPC client to fail instantly
+		ddConfig.Datadog.Set("cmd_port", "0")
```

Comment:
0 is a valid port number, indicating that the kernel should pick an unused port.  Especially since this is a string, perhaps it's best to set it to `-1` here?  If something parses it as a Uint16, that may cause an error, but otherwise it seems a better "not valid" semaphore for this case.